src/jsDelivr.js

```diff
@@ -12,16 +14,31 @@ function formatHits(pkg) {
 }
 
 export async function loadHits() {
-  const hitsJSONpromise = got(c.jsDelivrHitsEndpoint, { json: true });
-  const hitsJSON = (await hitsJSONpromise).body;
-  hits.clear();
-  hitsJSON.forEach(formatHits);
+  const start = Date.now();
+  log.info('📦  Loading hits from jsDelivr');
+
+  try {
+    const { body: hitsJSON } = await got(c.jsDelivrHitsEndpoint, {
+      json: true,
+    });
+    hits.clear();
+    hitsJSON.forEach(formatHits);
+  } catch (e) {
+    log.error(e);
+
+    // eslint-disable-next-line no-process-exit
+    process.exit(1);
```

Comment:
I've actually normally not crashed if jsDelivr failed earlier on purpose, because it's not the most important part of data. If it fails, the rest can still continue

Reply:
There was no `catch` before so I interpreted it that way. I think it was prone to crash because the error (if any) would bubble up. I'll remove the process.exit 👌